pkg/settings/setting.go

```diff
@@ -98,7 +98,7 @@ var (
 	IgnoreNodeName                    = NewSetting("ignore-node-name", "") // nodes to ignore when syncing v1.node to v3.node
 	NoDefaultAdmin                    = NewSetting("no-default-admin", "")
 	RestrictedDefaultAdmin            = NewSetting("restricted-default-admin", "false") // When bootstrapping the admin for the first time, give them the global role restricted-admin
-	EKSUpstreamRefreshCron            = NewSetting("eks-refresh-cron", "*/5 * * * *")
```

Comment:
Leave this setting as-is

Reply:
Done

Reply:
Functionally its fine, but add comment at end of setting like it is for others.
Also looks like there are other changes here, lets revert those. HideLocalCluster setting should not change. The only change here should be new setting and added comment.

Reply:
ok, only two lines are touched